block.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "create-block/review-block",
+	"title": "Review Block",
+	"category": "widgets",
+	"icon": "smiley",
+	"description": "Example block written with ESNext standard and JSX support – build step required.",
```

Comment:
Use the following as description.

`Embed WordPress plugin reviews`.

Reply:
Fixed.